crates/projection/src/lib.rs

```diff
@@ -9,7 +9,7 @@ pub enum Error {
     TooClose,
     #[error("position is behind the camera")]
     BehindCamera,
-    #[error("the pixel position cannot be projected to the ground")]
+    #[error("the pixel position cannot be projected on the projection plane")]
```

Comment:
"to", "on"... "onto"? 😄 
![image](https://github.com/HULKs/hulk/assets/33729490/e56a0429-674e-43b8-9268-cf1910fc46b6)

Reply:
done